Makefile

```diff
@@ -74,16 +74,12 @@ api/%/zz_generated.deepcopy.go: bin/controller-gen $(DEEPCOPY_GEN_INPUTS)
 
 MANAGER_BIN_INPUTS=$(shell find ./controllers ./api ./pkg -name "*mock*" -prune -o -name "*test*" -prune -o -type f -print) main.go go.mod go.sum
 .PHONY: build
-build: binaries generate-mocks generate-deepcopy manifests release-manifests ## Build manager binary.
+build: binaries generate-deepcopy lint manifests release-manifests ## Build manager binary.
```

Comment:
why are we not generating mocks any more as part of this? Again, eks-a might not like this very much. We need to be careful about what changes are being made to the build scripts since they have a dependency

Reply:
> why are we not generating mocks any more as part of this?
We are.

On line 169 `generate-mocks: bin/mockgen generate-deepcopy pkg/mocks/mock_client.go ` generate mocks generates deepcopy as a prerequisite. You shouldn't have to change the EKS-A command. 

Please let me know if you do.